js/ui/lookup.js

```diff
@@ -331,6 +331,17 @@ var Lookup = DropDownList.inherit({
             */
             usePopover: false,
 
+            /**
+             * @name dxLookupOptions.dropDownButtonTemplate
+             * @inheritdoc
+             * @hidden
+             */
+
+            /**
+             * @name dxLookupOptions.showDropDownButton
+             * @inheritdoc
+             * @hidden
+             */
```

Comment:
Why hide this options for lookup?

Reply:
This option does not work in lookup because dxLookup uses it's own custom button